src/main/resources/templates/user.html

```diff
@@ -18,6 +18,12 @@ <h1>Your Login Details</h1>
   <form th:action="@{/logout}" method="post">
     <input type="submit" value="Logout" />
   </form>
+
+  <!-- Add this form in your HTML file -->
+  <form th:action="@{/user/getTokens}" method="get">
+    <input type="submit" value="Get Refresh Token" />
+  </form>
+
```

Comment:
Can we have a section to fetch and display userinfo from the `userinfo` endpoint with the opaque access token?

Reply:
The code is in the master already. OAuth2.client will auto fetch userInfo once authenticated.